provisioning/miniprov/hare_mp/main.py

```diff
@@ -287,11 +287,23 @@ def save(filename: str, contents: str) -> None:
         f.write(contents)
 
 
+def generate_config(path_to_cdf: str):
+    conf_dir='/var/lib/hare'
+    os.environ['PATH'] += os.pathsep + '/opt/seagate/cortx/hare/bin/'
+    process = subprocess.run(['cfgen', '-o', conf_dir, path_to_cdf],
```

Comment:
There is `execute` function in this file. Why wouldn't you reuse it?

Reply:
Yes, thanks for pointing it out!

Reply:
Done